src/unix/linux-core.c

```diff
@@ -560,7 +583,7 @@ int uv_cpu_info(uv_cpu_info_t** cpu_infos, int* count) {
   *cpu_infos = NULL;
   *count = 0;
 
-  numcpus = sysconf(_SC_NPROCESSORS_ONLN);
+  numcpus = uv__cpu_num();
```

Comment:
This will fail terribly when /proc isn't mounted, the asserts below won't catch the error unless it's -EPERM.